### PR TITLE
feat: basic neurons page

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

- define the style for the `<section/>` of the app
- implement the basic content of the "neurons" page (title + text + display of principal id)

# Changes

- global style modification for app "signed in" content(background, positioning and spacing)
- small navigation height (closer to Flutter design)
- push up global font sizes (closer to Flutter design on large screen)
- implements "neurons" page

### CODE REVIEW

**Comment by nmattia** [comment]
Thanks! Do you have some screenshots we could look at?

**Comment by peterpeterparker** [comment]
@nmattia 

Capture d’écran 2022-01-13 à 14 17 36

File: frontend/svelte/src/lib/components/Layout.svelte [hunk 1]
```diff
@@ -7,4 +7,18 @@
...
 <Nav />
 
-<slot />
+<main>
+  <slot />
+</main>
+
+<style lang="scss">
+  main {
+    position: absolute;
```
Comment (nmattia) [inline frontend/svelte/src/lib/components/Layout.svelte:16]: I'm confused, why's everything absolute all of a sudden?

File: frontend/svelte/src/routes/Neurons.svelte [hunk 2]
```diff
@@ -1,16 +1,36 @@
 <script lang="ts">
   import Layout from "../lib/components/Layout.svelte";
-  import { onMount } from "svelte";
+  import { onDestroy, onMount } from "svelte";
+  import type { Unsubscriber } from "svelte/types/runtime/store";
+  import { AuthStore, authStore } from "../lib/stores/auth.store";
 
   onMount(() => {
```
Comment (nmattia) [inline frontend/svelte/src/routes/Neurons.svelte:7]: I'm guessing this should go away, correct?

EDIT: Ah, looks like this isn't the final page.

**Review by nmattia — COMMENTED** [review]
Two questions:

* Why did everything become `position: absolute`?
* Shouldn't we wait for the rest of the page before merging?

File: frontend/svelte/src/lib/components/Layout.svelte (hunk 1 above)
Comment (peterpeterparker) [reply frontend/svelte/src/lib/components/Layout.svelte:16]: Only the content should scroll. First time we implement content.

File: frontend/svelte/src/routes/Neurons.svelte (hunk 2 above)
Comment (peterpeterparker) [reply frontend/svelte/src/routes/Neurons.svelte:7]: Yes it will needs a footer and the actual neurons actions.

**Review by nmattia — APPROVED** [review]

**Comment by nmattia** [comment]
LGTM, thanks!

**Pull request merged**